addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -341,6 +350,17 @@ class Composer extends Component {
         this._postMessage();
     }
 
+    _setTextInputHtmlContent(textInputContent) {
+        //Removing unwanted extra spaces from message
+        let value = _.escape(textInputContent).trim();
+        value = value.replace(/(\r|\n){2,}/g, '<br/><br/>');
+        value = value.replace(/(\r|\n)/g, '<br/>');
```

Comment:
I'm not sure to understand why a second replace here?

Reply:
We'll see that later